go.mod

```diff
@@ -4,13 +4,16 @@ go 1.13
 
 require (
 	github.com/alessio/shellescape v0.0.0-20190409004728-b115ca0f9053
+	github.com/evanphx/json-patch v4.5.0+incompatible
 	github.com/google/uuid v1.1.1
+	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/pkg/errors v0.8.1
 	github.com/spf13/cobra v0.0.3
+	github.com/spf13/pflag v1.0.5 // indirect
 	golang.org/x/crypto v0.0.0-20190611184440-5c40567a22f8
+	gopkg.in/yaml.v2 v2.2.4 // indirect
```

Comment:
sigs.k8s.io/yaml, eventually we may de-dupe this if that library gets on .v3 but it's not looking likely soon